--- chromium/third_party/abseil-cpp/absl/debugging/failure_signal_handler.cc.orig	2021-08-13 12:36:58.000000000 +0200
+++ chromium/third_party/abseil-cpp/absl/debugging/failure_signal_handler.cc	2021-08-18 22:04:02.165382504 +0200
@@ -135,7 +135,7 @@
 #else
   const size_t page_mask = sysconf(_SC_PAGESIZE) - 1;
 #endif
-  size_t stack_size = (std::max(SIGSTKSZ, 65536) + page_mask) & ~page_mask;
+  size_t stack_size = (std::max<size_t>(SIGSTKSZ, 65536) + page_mask) & ~page_mask;
 #if defined(ABSL_HAVE_ADDRESS_SANITIZER) || \
     defined(ABSL_HAVE_MEMORY_SANITIZER) || defined(ABSL_HAVE_THREAD_SANITIZER)
   // Account for sanitizer instrumentation requiring additional stack space.
--- chromium/third_party/breakpad/breakpad/src/client/linux/handler/exception_handler.cc.orig	2021-08-18 22:05:45.366849996 +0200
+++ chromium/third_party/breakpad/breakpad/src/client/linux/handler/exception_handler.cc	2021-08-18 22:05:57.647024518 +0200
@@ -138,7 +138,7 @@
   // SIGSTKSZ may be too small to prevent the signal handlers from overrunning
   // the alternative stack. Ensure that the size of the alternative stack is
   // large enough.
-  static const unsigned kSigStackSize = std::max(16384, SIGSTKSZ);
+  static const unsigned kSigStackSize = std::max<size_t>(16384, SIGSTKSZ);
 
   // Only set an alternative stack if there isn't already one, or if the current
   // one is too small.
