--- qtwebengine-everywhere-src-5.11.1/mkspecs/features/platform.prf~	2018-06-15 11:13:33.000000000 +0200
+++ qtwebengine-everywhere-src-5.11.1/mkspecs/features/platform.prf	2018-08-11 23:35:10.234741469 +0200
@@ -12,11 +12,6 @@
 defineTest(isPlatformSupported) {
   QT_FOR_CONFIG += gui-private
   linux {
-    if(!gcc:!clang)|intel_icc {
-      skipBuild("Qt WebEngine on Linux requires clang or GCC.")
-      return(false)
-    }
-    gcc:!clang:!isGCCVersionSupported(): return(false)
   } else:win32 {
     winrt {
       skipBuild("WinRT is not supported.")
